.github/workflows/daily.yml

```
@@ -672,13 +672,34 @@ jobs:
         if: true && !contains(github.event.inputs.skiptests, 'unittest')
         run: ./src/valkey-unit-tests --accurate
 
-  test-centos7-jemalloc:
-    runs-on: ubuntu-latest
+  test-rpm-distros-jemalloc:
     if: |
-      (github.event_name == 'workflow_dispatch' || (github.event_name != 'workflow_dispatch' && github.repository == 'valkey-io/valkey')) &&
-      !contains(github.event.inputs.skipjobs, 'centos')
-    container: centos:7
+       (github.event_name == 'workflow_dispatch' || (github.event_name != 'workflow_dispatch')) &&
```

```suggestion
       (github.event_name == 'workflow_dispatch' || (github.event_name != 'workflow_dispatch' && github.repository == 'valkey-io/valkey')) &&
```
We use this to prevent kicking off daily runs from the forked repositories.